Makefile

```diff
@@ -48,7 +48,7 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr htmlcov/
 
 lint: ## check style with flake8
-	flake8 bigchaindb tests
+	flake8 --max-line-length=119 bigchaindb tests
```

Comment:
I think there's a project-level flake8 config somewhere (maybe `setup.cfg`?) that we could use so we don't have to use `--max-line-length=119` everywhere.